### PR TITLE
Add sections for hacker traffic logs to customer gateway page

### PR DESCRIPTION
**Pull request by saida**
Added two sections to https://docs.hackerone.com/organizations/hackerone-vpn.html 
- Download Hacker Traffic Logs
- Sync Hacker Traffic Logs
- 
Screenshot 2023-10-19 at 17 25 04

Added a new page for Gateway Hacker Traffic Logs.

Screenshot 2023-10-19 at 18 54 49

### CODE REVIEW

File: docs/organizations/hackerone-vpn.md [hunk 1]
```diff
@@ -41,12 +41,32 @@ You can pause or resume access of individual hackers by:
...
-The activity logs show what actions are being performed by Hackers while on the Gateway solution. It is possible to customize the results by using the filters available, such as **Date**, **Month**, **Year** and **start-end** time, or by using the search box.
+The activity overview shows what actions are being performed by Hackers while on the Gateway solution. It is possible to customize the results by using the filters available, such as **Date**, **Month**, **Year** and **start-end** time, or by using the search box.
 Please note that it can take up to 20 minutes to display the latest activity logs. 
 
-![gateway_activity_logs](./images/gateway-7.png)
+![gateway_activity_overview](./images/gateway-7.png)
+
+### Download Hacker Traffic Logs
+
+The log page allows you to download verbose logs of your network activities for the selected date at UTC.
```
Comment (jessiwright) [inline docs/organizations/hackerone-vpn.md:53]: Spell out UTC in parenthesis after

File: docs/organizations/hackerone-vpn.md [hunk 2]
```diff
@@ -41,12 +41,32 @@ You can pause or resume access of individual hackers by:
...
+
+The log page allows you to download verbose logs of your network activities for the selected date at UTC. 
+It might take some time to generate logs depending on the hacker traffic volume on the given date.
+You will be notified by email once the logs are ready to download. 
+
+The links to download logs are valid for 24 hours. When expired, you can request logs for the same date again and 
+a new link will be generated quickly. You can request a maximum of 5 logs at the same time.
+
+You can view all log requests that have been made for the given program. The table shows request details such as 
+request time, status, requester, the date the log requested for, expiration time of the download link and
```
Comment (jessiwright) [inline docs/organizations/hackerone-vpn.md:61]: remove extra line break in the middle of the sentence
"the date the log **was** requested for"

File: docs/organizations/hackerone-vpn.md (hunk 1 above)
Comment (saida) [reply docs/organizations/hackerone-vpn.md]: Updated!

File: docs/organizations/hackerone-vpn.md [hunk 3]
```diff
@@ -41,12 +41,32 @@ You can pause or resume access of individual hackers by:
...
+![gateway_activity_overview](./images/gateway-7.png)
+
+### Download Hacker Traffic Logs
+
+The log page allows you to download verbose logs of your network activities for the selected date at UTC. 
+It might take some time to generate logs depending on the hacker traffic volume on the given date.
+You will be notified by email once the logs are ready to download. 
+
+The links to download logs are valid for 24 hours. When expired, you can request logs for the same date again and 
+a new link will be generated quickly. You can request a maximum of 5 logs at the same time.
```
[the remark this answers is not in the document]
Comment (saida) [reply docs/organizations/hackerone-vpn.md]: Removed!

File: docs/organizations/hackerone-vpn.md (hunk 2 above)
Comment (saida) [reply docs/organizations/hackerone-vpn.md]: Updated!

**Comment by saida** [comment]
I've added a new page for Hacker Traffic Logs as well.

image

@jessiwright could you review again please?

Thanks!

**Review by jessiwright — APPROVED** [review]

**Pull request merged**